dist/images/install.sh

```diff
@@ -643,6 +643,30 @@ spec:
                   type: boolean
                 ipv6RAConfigs:
                   type: string
+                acls:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      direction:
+                        type: string
+                        enum:
+                          - from-lport
+                          - to-lport
+                      priority:
+                        type: integer
+                        minimum: 0
+                        maximum: 32767
+                      match:
+                        type: string
```

Comment:
If `match` must not be empty, we can add it to `required`. Example: https://github.com/kubeovn/kube-ovn/blob/master/yamls/crd.yaml#L278.

Reply:
This is the simplest implementation for acls. Maybe we can build the match expression automatically in the future, in which case the match field can be nil.